### DISCUSSION TITLE
Crashed with Unhandled Exception: System.IO.FileNotFound.Exception

### DISCUSSION THREAD

**Discussion by chiyu1203** [bonsai-rx/bonsai#1366] [Q&A]
Our laptop was recently upgraded to Windows 11 and I started to see weird things when I ran Bonsai. Therefore I wonder if anyone can give me some ideas.  
I can run Bonsai as usual by double clicking the programme or the shortcut and then load a project (an existing workflow) from the GUI.
However, if I double clicks a Bonsai project, I will see an error about Crashed with Unhandled Exception: System.IO.FileNotFound.Exception
And then the programme shut down by itself
![Screenshot_20230518-121724](https://github.com/bonsai-rx/bonsai/assets/16696523/8b06cd4f-36c9-4ea0-ba18-cefafec5a7d7)
 (see attached picture)

Does anyone know what might happen? 

Below is my software and device specifications
Bonsai version: 2.7.2, installed via Portable (.zip)
Device Specifications:
Intel i7-1185G7
RAM 32.0GB
Windows 11 64-bit

Many thanks in advance!

**Comment by bruno-f-cruz** [comment]
Delete and reinstall the system's bonsai, should fix it.

**Reply by chiyu1203** [reply]
I will give it a try! thanks!
